site/package.json

```diff
@@ -10,12 +10,13 @@
 	"dependencies": {
 		"@banhmi/core": "1.0.0-rc1",
 		"@banhmi/icon": "0.13.0",
-		"@moai/core": "1.0.0-rc7",
+		"@moai/core": "1.0.0-rc10",
+		"@moai/gallery": "1.0.0-rc11",
 		"@types/node": "14.14.35",
 		"@types/react": "17.0.3",
 		"next": "10.0.9",
 		"react": "17.0.2",
 		"react-dom": "17.0.2",
-		"typescript": "4.2.3"
+		"typescript": "4.2.3",
```

Comment:
This is weird, the last item should not have a comma I think?

Reply:
yes :(